util/fork_func.c

```diff
@@ -12,7 +12,7 @@
  * Fork and run ${func} in a new process, with ${cookie} as the sole argument.
  */
 pid_t
-fork_func(int (* func)(void *), void * cookie)
+fork_func(int (*func)(void *), void * cookie)
```

Comment:
wait, now I'm rethinking this one.

in `events/*.c`, there's 6 function pointers defined as `(*func)` with no spaces.  There's only 2 instances of `(* func)`, both of which in cpusupport code I wrote.

OTOH, when I grep for `(* `, there's 56 hits, and at first glance, at least 90% of them are function pointers.

So I'm guessing that the desired style is:
```
int
foo(int (* func)(void *))
```
?

Reply:
Yes, our STYLE says "space between * and variable name".  Applies to function pointers just as much as any other pointers.

Reply:
I removed the bad commit, and made a separate PR for fixing the style of our function pointers.